perma_web/perma/tests/test_views_user_management.py

```diff
@@ -712,6 +712,7 @@ def initialize_form(csv_file, data=None):
         one_more_valid_csv_file = create_csv_file('one_more_valid_users.csv', csv_data)
         invalid_csv_file = create_csv_file('invalid_users.csv', invalid_csv_data)
         another_invalid_csv_file = create_csv_file('another_invalid_users.csv', another_invalid_csv_data)
+        one_more_invalid_csv_file = create_csv_file('users.csv', csv_data, 'utf-16')
```

Comment:
It's minor, but maybe we could name this more descriptively since it's a different kind of invalidity? e.g.

```suggestion
        csv_file_with_invalid_encoding = create_csv_file('users.csv', csv_data, 'utf-16')
```